test/components/center_test.dart

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
This is actually the only thing that keeps it from being merged. Please implement the test before you work on any other PR. Thank you @armantorkzaban

Reply:
The test is already there, implemented. Needs Bbox though, in order to work.

Reply:
I think at this point we can merge the two branches and deal with them as one. @lukas-h, what do you think?

Reply:
We could also try to combine the two branches to one

Reply:
What do you mean by 'combining'? Do you mean here on GH or do you mean merging? or both?